src/Types/StructuralTypes/StructuralType.php

```diff
@@ -40,32 +43,51 @@
  * - relationship
  * - path
  *
- * @note This interface should not be implemented by any class directly.
```

Comment:
Why was this comment removed?

Reply:
Because some methods had to be implemented by classes directly